packages/flow-client/scripts/generate-models.ts

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env tsx
+
+import { dirname } from 'node:path';
+import {
+  typeScriptDefaultModelNameConstraints,
+  TypeScriptGenerator,
+} from '@asyncapi/modelina';
+import { readFile, writeFile, mkdir } from 'node:fs/promises';
+import { rm } from 'node:fs/promises';
+import { parse } from 'yaml';
+import { exec } from 'node:child_process';
+
+const packageDir = `${dirname(require.main?.filename ?? '')}/..`;
+const rootDir = `${packageDir}/../..`;
+
+function checkNodeVersion() {
+  const version = process.version;
+  const major = Number.parseInt(version.split('.')[0]);
+  if (major < 22) {
+    throw new Error('Node.js version 22 or higher is required');
+  }
+}
+
+const generator = new TypeScriptGenerator({
+  constraints: {
+    modelName: typeScriptDefaultModelNameConstraints({
+      NAMING_FORMATTER: (name) => {
+        if (name === 'Error') {
+          return 'ErrorType';
+        }
+        return name;
+      },
+    }),
+  },
+
+  modelType: 'interface',
+  enumType: 'union',
+  mapType: 'indexedObject',
+  rawPropertyNames: true,
+  moduleSystem: 'ESM',
```

Comment:
This will fix the pipeline

```suggestion
  moduleSystem: 'ESM',
  typeMapping: {
    // Anywhere we would output `any` we want to output `unknown` instead
    Any: () => {
      return 'unknown';
    },
  },
```